### PR TITLE
Added info about second factor to offline operations docs

### LINKED ISSUE #32: Improve Offline Authorization documentation
Add section, where we explain that the QR operations are 2FA only and the biometry is allowed based on the property in `flags`.

### PR DESCRIPTION
**Pull request by kober32**
Fixed #32

### CODE REVIEW

**Comment by kober32** [comment]
Tests will fail until #46 will be resolved

**Review by TomasKypta — APPROVED** [review]

File: docs/Using-Operations-Service.md [hunk 1]
```diff
@@ -170,14 +170,26 @@ After that, you can produce an off-line signature using the following code:
...
-    authentication.usePossession = true
-    authentication.usePassword = password
-    return operationsService.authorizeOfflineOperation(operation, authentication)
+fun approveQROperation(operation: QROperation, password: String?): String {
+    val auth = PowerAuthAuthentication()
+    auth.usePossession = true
+    if (password != null) {
+        // if a password is passed, use it as the second factor
+        auth.usePassword = password
+    } else if (operation.flags.biometryAllowed) {
```
Comment (petrdvorak) [inline docs/Using-Operations-Service.md:179]: I don't like this example very much. Could we explain the logic in a series of simpler examples?

1. Check if the biometry is allowed for offline operation.
2. If yes, show the biometric approval.
3. If no, ask user for password and use a password approval.

**Review by petrdvorak — APPROVED** [review]
I don't like something on that example - it assumes too much logic around it and uses side-effect to trigger biometric authentication, potentially resulting on logical error. Could we improve it by explaining what needs to happen overall?

File: docs/Using-Operations-Service.md (hunk 1 above)
Comment (kober32) [reply docs/Using-Operations-Service.md]: Improved in https://github.com/wultra/mtoken-sdk-android/pull/45/commits/4fd610e503b9ae1438f9982c1caccdc34293f03c

**Pull request merged**